### DISCUSSION TITLE
APP_NAME is not changing

### DISCUSSION THREAD

**Discussion by neon010** [outline/outline#5046] [General] [answered]
I am trying to change the APP_NAME from server/env.ts file to my app name. It is not working in production environment.
I am using bulding local docker image from dockerfile and running the project using docker-compose file.

Below is my docker-compose file
`
version: "3"
services:
  outline:
    image: outline-local:lattest
    env_file: ./docker.env
    ports:
      - "3002:3002"
    depends_on:
      - postgres
      - redis

  redis:
    image: redis
    env_file: ./docker.env
    ports:
      - "6379:6379"
    volumes:
      - ./redis.conf:/redis.conf
    command: [ "redis-server", "/redis.conf" ]
    healthcheck:
      test: [ "CMD", "redis-cli", "ping" ]
      interval: 10s
      timeout: 30s
      retries: 3

  postgres:
    image: postgres
    env_file: ./docker.env
    ports:
      - "5432:5432"
    volumes:
      - database-data:/var/lib/postgresql/data
    healthcheck:
      test: [ "CMD", "pg_isready" ]
      interval: 30s
      timeout: 20s
      retries: 3
    environment:
      POSTGRES_USER: 'user'
      POSTGRES_PASSWORD: 'pass'
      POSTGRES_DB: 'outline'
  s3:
    image: lphoward/fake-s3
    ports:
      - "4569:4569"
    volumes:
      - ./fakes3:/fakes3_root  
volumes:
  database-data:
`

**Comment by tommoor** [comment, marked as answer]
`APP_NAME` is not supported in the community edition, this is part of the paid edition. You can obtain a license by contacting hello@getoutline.com
